header/headertest/verify_test.go

```diff
@@ -37,14 +37,6 @@ func TestVerify(t *testing.T) {
 			},
 			err: true,
 		},
-		{
```

Comment:
this case is now covered by header.Validate instead which does basic sanity check on Commit

Reply:
However it is NOT done inside of Verify so it is important that Validate + Verify are called on every header